Formula/logstash.rb

```diff
@@ -9,6 +9,13 @@ class Logstash < Formula
     depends_on :java => "1.8"
   end
 
+  devel do
+    url "https://download.elastic.co/logstash/logstash/logstash-5.0.0-alpha2.tar.gz"
+    sha256 "575615d7d0e6b5dbf8805dc0a7420753e093ddd6f14cbe3229f73fb5eae14d80"
+    version "5.0.0-alpha2"
+    depends_on :java => "1.8"
+  end
```

Comment:
This block should be placed before `head do`. Use `brew audit --strict logstash` next time. :wink: